examples/tpm2_filternv/tpm2_filternv

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/python3
+
+import argparse
+import os
+from binascii import hexlify
+from tpm2_pytss.ESAPI import ESAPI
+from tpm2_pytss.TCTILdr import TCTILdr
+from tpm2_pytss.constants import TPMA_NV, TPM2_CAP, TPM2_HC
+
+
+output_template = """{pub.nvIndex:#x}:
+  name: {hexname}
+  hash algorithm:
+    friendly: {pub.nameAlg}
+    value: {pub.nameAlg:#x}
+  attributes:
+    friendly: {pub.attributes}
+    value: {pub.attributes:#x}
+  size: {pub.dataSize}
+"""
+
+
+def get_tcti(arg):
+    if arg:
+        return TCTILdr.parse(arg)
+    elif os.getenv("PYTSS_EXAMPLE_TCTI"):
+        return TCTILdr.parse(os.getenv("PYTSS_EXAMPLE_TCTI"))
+    return TCTILdr()
+
+
+def tpma_nv_arg(arg):
+    return TPMA_NV.parse(arg)
```

Comment:
(Suggestion) This could accept integers too:

```python
def tpma_nv_arg(arg):
    try:
        return TPMA_NV.parse(arg)
    except ValueError:
        # Decode an integer
        return TPMA_NV(int(arg, 0))
```

This way, using `./tpm2_filternv -F 0x10` could work, for example. What do you think?
(Or `parse` could be extended to support integers such as `"1"` and `"0x10"`)

Reply:
Good idea, added a variation of that